ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js

```diff
@@ -24,52 +28,66 @@ export default function AdvancedGasControls() {
         titleDetailText=""
         value={gasLimit}
       />
-      <AdvancedGasControlsRow
-        titleText={t('maxPriorityFee')}
-        tooltipText=""
-        onChange={setMaxPriorityFee}
-        value={maxPriorityFee}
-        titleDetailText={
-          <>
-            <Typography
-              tag="span"
-              color={COLORS.UI4}
-              variant={TYPOGRAPHY.H8}
-              fontWeight={FONT_WEIGHT.BOLD}
-            >
-              {t('gasFeeEstimate')}:
-            </Typography>{' '}
-            <Typography
-              tag="span"
-              color={COLORS.UI4}
-              variant={TYPOGRAPHY.H8}
-            ></Typography>
-          </>
-        }
-      />
-      <AdvancedGasControlsRow
-        titleText={t('maxFee')}
-        tooltipText=""
-        onChange={setMaxFee}
-        value={maxFee}
-        titleDetailText={
-          <>
-            <Typography
-              tag="span"
-              color={COLORS.UI4}
-              variant={TYPOGRAPHY.H8}
-              fontWeight={FONT_WEIGHT.BOLD}
-            >
-              {t('gasFeeEstimate')}:
-            </Typography>{' '}
-            <Typography
-              tag="span"
-              color={COLORS.UI4}
-              variant={TYPOGRAPHY.H8}
-            ></Typography>
-          </>
-        }
-      />
+      {process.env.SHOW_EIP_1559_UI ? (
+        <>
+          <AdvancedGasControlsRow
+            titleText={t('maxPriorityFee')}
+            tooltipText=""
```

Comment:
[Q] Are we just waiting for tooltip copy here?

Reply:
Yes -- I didn't see copy for that in the Figma file.